saleor/core/utils/random_data.py

```diff
@@ -1789,6 +1790,29 @@ def create_pages():
         yield f"Page {page.slug} created"
 
 
+def create_site_settings():
+    types = get_sample_data()
+    data = types["site.sitesettings"]
+
+    fk_fields = [
+        "site",
+        "top_menu",
+        "bottom_menu",
+        "refund_reason_reference_type",
+    ]
+
+    for settings_item in data:
+        pk = settings_item["pk"]
+        defaults = dict(settings_item["fields"])
+
+        for field in fk_fields:
+            if field in defaults:
+                defaults[f"{field}_id"] = defaults.pop(field)
```

Comment:
Do we really need this logic? Instead we can adjust `populatedb_data.json`, or just provide field values without `_id` at the end, I think that django will handle it properly.